docs/configuration/sinks/kafka.rst

```diff
@@ -16,6 +16,21 @@ Configuring the Kafka sink
             kafka_url: "localhost:9092"
             topic: "robusta-playbooks"
 
+.. admonition:: Add this to your generated_values.yaml if configuring with authentication
```

Comment:
I would add the link to KafkaProducer docs on fields available for the __init__
Maybe to rename `auth` to something like `kwargs`? I am not sure on this one tho

Reply:
I was also conflicted here and went back and forth a few times with something like `additonalConfigs`, `kwargs` ect.

I think spelling out the auth and the link to the different auth types is the most direct way to resolve questions with how do you connect kafka. If its too generic I can see people getting confused where they put the auth.